src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml

```diff
@@ -12,7 +12,29 @@
     </div>
 </div>
 
-<main class="govuk-main-wrapper dashboard" id="main-content" role="main">
+<section class="das-section das-section--search">
+    <div class="govuk-width-container">
+        <div class="das-dashboard-header__row">
+            <div class="das-dashboard-header__col das-dashboard-header__col--primary">
+                <form>
+                    <div class="govuk-form-group das-form-group-inline das-dashboard-search">
+                        <label for="search" class="govuk-visually-hidden">Search</label>
+                        <input class="govuk-input das-dashboard-search__input" id="search" type="search" placeholder="Search" disabled="disabled">
+                        <div class="das-form-group-inline__button-wrap">
+                            <button class="govuk-button das-dashboard-search__button" type="submit">Search</button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="das-dashboard-header__col das-dashboard-header__col--secondary">
+                <h2 class="das-placeholder-tasks">Other tasks you can do</h2>
+            </div>
+        </div>
+    </div>
+</section>
+
+
+<main class="govuk-main-wrapper das-section--dashboard" id="main-content" role="main">
     <div class="govuk-width-container">
         <div class="das-panels">
             <div class="das-panels__row">
```

Comment:
Should be  <div class="das-panels__col das-panels__col--primary">

Reply:
👍